authors/apps/articles/serializers.py

```diff
@@ -2,17 +2,20 @@
 
 from authors.apps.profiles.serializers import ProfileSerializer
 
-from .models import Article, Favorite, Like, Snapshot, ThreadedComment
+from .models import Article, Favorite, Like, Snapshot,\
+    ThreadedComment, Rating
```

Comment:
would you use parenthesis here instead?
```suggestion
  from .models import (Article, Favorite, Like, Snapshot,
    ThreadedComment, Rating)
```

Reply:
I have corrected this.